src/carousel.coffee

```diff
@@ -81,6 +81,11 @@ Ember.Widgets.CarouselComponent = Ember.Component.extend
         $active.removeClass(['active', direction].join(' '))
         @set 'sliding', no
 
+        # Send an action out to signal that the sliding transition has ended
+        # It can is useful to avoid having observers in any parentView having
+        # this component.
+        @send 'transitionEnded'
```

Comment:
When we add any actions, we should document when the action is triggered, and what data it is triggered with. It is part of the public API of this component. 

Our comments should assume that the people looking at the source have a basic literacy of Ember and it's concepts.